models/commons/src/index.ts

```diff
@@ -2,4 +2,6 @@
 export * from './localized-string/types';
 
 // Export models
+export * as Initiator from './initiator';
```

Comment:
What model is this exactly? Can we use a more specific name maybe?

Reply:
Initiator is used for `lastModifiedBy` and `createdBy` fields
https://docs.commercetools.com/api/types#client-logging

Reply:
Hmm ok, but I'm still confused about this name. I can't find it anywhere in the docs. Shouldn't this be called `ClientLogging` or something like that? 🤔

Reply:
As discussed, I added a description and code comment referring to the docs